syft/pkg/cataloger/cataloger.go

```diff
@@ -44,6 +45,7 @@ func ImageCatalogers(cfg Config) []Cataloger {
 		java.NewJavaCataloger(cfg.Java()),
 		apkdb.NewApkdbCataloger(),
 		golang.NewGoModuleBinaryCataloger(),
+		dotnet.NewDotnetDepsCataloger(),
```

Comment:
Question: does the `*.deps.json` file positively indicate installed software? or is it mostly indicative of dependencies that you would find in a git repo?

This may seem like an odd question, but let me provide a little context. The image catalogers only raise up packages which have a high confidence of being installed. So for example, a ruby `*.gemspec` file alone may be an indication of an installed package or could be found in the root of a git repo and there is no software "installed". The way we distinct this is by ensuring that the gemspec is within a "specifications" directory, which is where gemspecs are found when they have been installed by the ruby toolchain (`**/specifications/**/*.gemspec`).

I'm trying to understand if the `.deps.json` falls into one or both categories. Does it indicate declared dependencies (e.g. in a source repo)? Does it indicate installed software? Or both? (and if both, can we distinct the two cases clearly enough, say as we do with gemspecs?)

Thanks in advance!

Reply:
The *.deps.json file is generated from the compiler, so you won't find that file in a git-repo as it would be gitignored. Does this answer the question?

Reply:
I think it does yeah --it's distinct from nuget (a package manager which would be installations) and distinct from a declared project manifest (which would be outside install cases) but still seems strongly associated with something that has been compiled and run.

Thanks!